assets/src/scripts/components/hydrograph/index.js

```diff
@@ -13,6 +13,7 @@ import {Actions as ivTimeSeriesDataActions} from '../../store/instantaneous-valu
 import {Actions as ivTimeSeriesStateActions} from '../../store/instantaneous-value-time-series-state';
 import {Actions as statisticsDataActions} from '../../store/statistics-data';
 import {Actions as timeZoneActions} from '../../store/time-zone';
+import {Actions as floodDataActions} from '../../store/flood-inundation';
```

Comment:
Since the state and data parts of the actions are all in flood-inundation, why don't we call these floodActions. We are inconsistent here. It's called floodDataActions here and floodStateActions in the testing file. Or stick with floodStateActions. But it should be consistent.

Reply:
I changed the spec file to be floodData actions to be consistent.